posthog/api/cohort.py

```diff
@@ -617,6 +617,29 @@ def persons(self, request: Request, **kwargs) -> Response:
 
         return Response({"results": serialized_actors, "next": next_url, "previous": previous_url})
 
+    @action(methods=["PATCH"], detail=True)
+    def add_persons_to_static_cohort(self, request: request.Request, **kwargs):
+        cohort: Cohort = self.get_object()
+        if not cohort.is_static:
+            raise ValidationError("Can only add users to static cohorts")
+        person_ids = request.data.get("person_ids", None)
+        if not isinstance(person_ids, list):
+            raise ValidationError("person_ids need to be a list")
+        if len(person_ids) == 0:
+            raise ValidationError("Cannot have empty person_ids")
+        if len(person_ids) >= DEFAULT_COHORT_INSERT_BATCH_SIZE:
+            raise ValidationError("List size exceeds limit")
+        uuids = [
+            str(uuid)
+            for uuid in Person.objects.db_manager(READ_DB_FOR_PERSONS)
+            .filter(team_id=self.team_id, uuid__in=person_ids)
+            .values_list("uuid", flat=True)
+        ]
+        if len(uuids) == 0:
+            raise ValidationError("No valid users to add to cohort")
+        cohort.insert_users_list_by_uuid(uuids, team_id=self.team_id, insert_in_clickhouse=True)
```

Comment:
Not sure if this should be inserting into clickhouse

Reply:
What does uploading CSV files do? This should do the same.